pupil_src/shared_modules/surface_tracker/surface_tracker_offline.py

```diff
@@ -34,6 +34,13 @@
 from surface_tracker.gui import Heatmap_Mode
 from surface_tracker.surface_offline import Surface_Offline
 
+# On macOS, "spawn" is set as default start method in main.py. This is not required
+# here and we set it back to "fork" to improve performance.
+if platform.system() == "Darwin":
+    mp = multiprocessing.get_context("fork")
```

Comment:
nitpick: since `mp` is frequently used as a shorthand for `multiprocessing`, it could be better to explicitly name this `mp_context` or `mp_default_context`

Reply:
Agreed

Reply:
I actually found an other mp-inconsistency thanks to the renaming. 👍